tests/queries/0_stateless/02354_parse_timedelta.sql

```diff
@@ -0,0 +1,7 @@
+SELECT parseTimeDelta('1 min 35 sec');
+SELECT parseTimeDelta('0m;11.23s.');
+SELECT parseTimeDelta('11hr 25min 3.1s');
+SELECT parseTimeDelta('0.00123 seconds');
+SELECT parseTimeDelta('1yr2mo');
+SELECT parseTimeDelta('11s+22min');
+SELECT parseTimeDelta('1yr-2mo-4w + 12 days, 3 hours : 1 minute ; 33 seconds');
```

Comment:
Let's also write tests with invalid expressions to verify that we handle them correctly.
You can write such tests like:
```sql
SELECT parseTimeDelta('1yyr'); -- { serverError BAD_ARGUMENTS }
```

Reply:
> Let's also write tests with invalid expressions to verify that we handle them correctly. You can write such tests like:
> 
> ```sql
> SELECT parseTimeDelta('1yyr'); -- { serverError BAD_ARGUMENTS }
> ```

When I write invalid expressions, command "../../../build/programs/clickhouse-client -n  < 02354_parse_timedelta.sql > 02354_parse_timedelta.reference" got exception and couldn't get expected result. So I just add invalid expressions and comment them.